pkg/util/util.go

```diff
@@ -45,6 +45,23 @@ func removeDuplicatedVolumeMounts(volumeMounts []corev1.VolumeMount) []corev1.Vo
 	return results
 }
 
+func mergeDNSPolicyAndHostNetwork(gPolicy, policy *corev1.DNSPolicy, gHostNetwork, hostNetwork *bool) {
+	if *gPolicy == "" {
+		*gPolicy = *policy
+	}
+
+	if !*gHostNetwork {
+		*gHostNetwork = *hostNetwork
+	}
+}
+
+func setDefaultDNSPolicy(gPolicy *corev1.DNSPolicy) corev1.DNSPolicy {
+	if *gPolicy == "" {
+		return "ClusterFirst"
```

Comment:
Why not keep it undefined and let Kubernetes' default apply?

Reply:
'Cos kubernetes default DNSPolicy is `ClusterFirst `. I will test it if fill it with empty string.

Reply:
If filled with empty string, DNSPolicy will be `Default`, not `ClusterFirst `. That is not default behavior for kubernetes DNSPolicy.

Reply:
If filled with empty string, DNSPolicy will be `ClusterFirst` . That is  default behavior for kubernetes DNSPolicy.

Reply:
I will remove this function